helm-chart/values.yaml

```diff
@@ -64,22 +67,36 @@ debug:
     useLocal: false
 
 worker:
-  useKedaAutoscaler: false
+  # These are the tolerations for the worker-deplotment.yaml that runs continuously
+  # on a specific node.
+  tolerations: [{key: "highMemory", operator: "Exists"}] # example: [{key: "exampleKey", operator: "Exists"}]
   preventPodsFromBeingScheduledOnSameNodeAsAPI: true
+
+  # If true, the worker will use KEDA to autoscale the number of pods based on the number of tasks in the queue.
+  useKedaAutoscaler: false
+
+  # The following values are used to set the configuration options for the autoscaling.
+  # Only used if useKedaAutoscaler is true.
   scaling:
     maxReplicaCount: 10
     scaledComponents:
       - identifier: "medium"
         preferredNodeType: ""
```

Comment:
No small ?
I recall we use two select between the 3, not only between two.
what will happen for small calculations ? they remains on keda triggering ?

Reply:
The small is not autoscaled using keda.